### DISCUSSION TITLE
Why does `dark` color scheme not work with `lightTheme`

### DISCUSSION THREAD

**Discussion by AlexMcComb** [adobe/react-spectrum#2456] [Q&A]
Why does this work for dark theme:
` <Provider theme={darkTheme} colorScheme="light">` & 
` <Provider theme={darkTheme} colorScheme="dark">`

but this does not work for light theme:
` <Provider theme={lightTheme} colorScheme="dark">` 

but this does?:
` <Provider theme={lightTheme} colorScheme="light">`

I'm using this for a CEP plugin, so I'm trying to match the Creative Cloud application UI "brightness" changes to match my plugin theme color.  Creative Cloud applications have 4 brightnesses I'd like to match.

**Comment by reidbarber** [comment]
The light theme should apply the `darkest` color scheme for `colorScheme="dark"`:

https://github.com/adobe/react-spectrum/blob/ab0323af3512117c9607304a66a33faf026929c9/packages/%40react-spectrum/theme-light/src/index.ts#L24-L25

The included themes support the following color schemes:

| Theme      | Color Scheme |
| ----------- | ----------- |
| `defaultTheme`      | `light` / `darkest`       |
| `darkTheme`   | `dark` / `darkest`        |
| `lightTheme`   | `lightest` / `darkest`        |

**Reply by AlexMcComb** [reply]
Ok, so I need the default theme for the light color scheme - thanks!
